Web/tues_pairs/src/components/Pages/Home/home.jsx

```diff
@@ -34,7 +38,15 @@ class UserList extends Component {
         matchedUserID: currentUser.matchedUserID
       }, {merge: true})
       .then(() => {
-        this.props.authUser.matchedUserID = currentUser.matchedUserID;
+        const users = this.state.users;
+        users.forEach(user => {
+          if(user.uid !== currentUser.matchedUserID && user.uid !== currentUser.uid && user.matchedUserID === currentUser.uid) {
```

Comment:
I would format this condition in different way. Put one condition or max two on each row.